internal/tools/bigquery/bigquerygetdatasetinfo/bigquerygetdatasetinfo.go

```diff
@@ -118,7 +118,7 @@ type Tool struct {
 	mcpManifest tools.McpManifest
 }
 
-func (t Tool) Invoke(ctx context.Context, params tools.ParamValues) (any, error) {
+func (t Tool) Invoke(ctx context.Context, params tools.ParamValues, accesToken tools.OAuthAccessToken) (any, error) {
```

Comment:
```suggestion
func (t Tool) Invoke(ctx context.Context, params tools.ParamValues, accessToken tools.OAuthAccessToken) (any, error) {
```
Update in other places as well

Reply:
updated